rockcraft/models/project.py

```diff
@@ -219,6 +222,28 @@ def effective_base(self) -> bases.BaseName:
     def model_reference_slug(cls) -> str | None:
         return "/reference/rockcraft.yaml"
 
+    @override
+    def get_build_plan(self) -> list[BuildInfo]:
+        platforms = typing.cast(
+            # https://github.com/canonical/craft-platforms/issues/43
```

Comment:
Since this is fixed, I think we can drop this with a release of craft-platforms.